### PR TITLE
Added note in Preferences section

### PR DESCRIPTION
**Pull request by lucamanga**
Added note in Preferences section, to remove settings in the old Preferences section otherwise it will not work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Policies can be specified using the [Group Policy templates on Windows](https://
 | **[`Permissions`](#permissions)** | Set permissions associated with camera, microphone, location, and notifications.
 | **[`PictureInPicture`](#pictureinpicture)** | Enable or disable Picture-in-Picture.
 | **[`PopupBlocking`](#popupblocking)** | Configure the default pop-up window policy as well as origins for which pop-up windows are allowed.
-| **[`Preferences`](#preferences)** | Set and lock preferences.
+| **[`Preferences`](#preferences)** | Set and lock preferences. (NOTE: in order to use this, settings in the **[`Preferences (Deprecated)`](#preferences-deprecated)** section must be cleared.
```
Comment (mkaply) [inline README.md:78]: Can you remove this one? They actually do mix and match everywhere except the GPO.

So I want to add a comment in the GPO itself.

File: README.md [hunk 2]
```diff
@@ -3752,6 +3752,8 @@ Value (string):
 ### Preferences
 Set and lock preferences.
 
+**NOTE** In order to use this policy, you must clear all settings in the old **Preferences (Deprecated)** section.
```
Comment (mkaply) [inline README.md:3755]: Can you make this GPO/Windows specific? In order to use this policy on Windows.

Thanks!

**Review by mkaply — CHANGES_REQUESTED** [review]

**Review by lucamanga — COMMENTED** [review]
Made edit as your suggestion, @mkaply

**Comment by mkaply** [comment]
Thanks!

**Comment by mkaply** [comment]
I made the changes in each ADMX.

https://github.com/mozilla/policy-templates/commit/3c123e54a10662848d105e0df2d639efb7138430

**Pull request merged**